pkg/build/buildkit/connector/wait.go

```diff
@@ -110,6 +122,13 @@ func (bw *Waiter) WaitUntilIsUp(ctx context.Context) error {
 			bw.waitingTime = bw.maxWaitTime
 			return fmt.Errorf("buildkit service not available for %v", bw.maxWaitTime)
 		default:
+			if err := bw.connectionManager.Start(ctx); err != nil {
+				bw.logger.Infof("Failed to start BuildKit service: %v\n", err)
```

Comment:
Nit (or to be addressed later). We are not starting BuildKit service, we are just starting a  connection with Buildkit